pools/hashalot.go

```diff
@@ -54,5 +54,14 @@ func (p *Hashalot) GetName() string {
 }
 
 func (p *Hashalot) GetFee() float64 {
-	return 1.00
+	jsonPayload := map[string]interface{}{}
+	err := util.GetJson(fmt.Sprintf("http://api.hashalot.net/pools/"), &jsonPayload)
+	if err != nil {
+		return 0
```

Comment:
I think you shouldn't return a 0 here, but whatever fee is current? I mean, 0% is probably wrong most times

Reply:
2.0% is current, will update to return at least that to be safe.  Do you also want me to cycle the default pools on this one since it's now more expensive?

Reply:
https://github.com/vertcoin-project/one-click-miner-vnext/commit/a3b980c11cce5c7e4be3ff797a69c766454706fd was merged which randomizes pool selection on first start